lib/web/fetch/body.js

```diff
@@ -403,14 +403,14 @@ function mixinBody (prototype) {
 async function consumeBody (object, convertBytesToJSValue, instance) {
   webidl.brandCheck(object, instance)
 
-  throwIfAborted(object[kState])
-
   // 1. If object is unusable, then return a promise rejected
   //    with a TypeError.
   if (bodyUnusable(object[kState].body)) {
     throw new TypeError('Body is unusable')
   }
 
+  throwIfAborted(object[kState])
```

Comment:
I'm certain this can also be removed. My guess is that when the stream is closed it'll fail in fullyReadBody.